tools/release-ci/pkg/config/batch.go

```diff
@@ -0,0 +1,56 @@
+package config
+
+import "fmt"
+
+const (
+	PullRequestPrefix = "pr-logs"
+	PullRequestDirectory = PullRequestPrefix + "/directory"
+)
+
+// Repo contains configuration for a published
+// repository version involved in a job
+type Repo struct {
+	// RepoOwner is the GitHub organization
+	// that triggered this build, provided as
+	// $REPO_OWNER by Prow.
```

Comment:
That was not clear from the Prow docs. The linked page just has a matrix of job type and parameter -- we should aim for that to be identical between Pods and Jenkins jobs IMO